src/createChart.js

```diff
@@ -18,7 +18,7 @@ export default function createChart(element = 'body', config = {}, controls = nu
 
     thisChart.marks = [];
 
-    thisChart.wrap = select(thisChart.div).append('div');
+    thisChart.wrap = select(thisChart.div).append('div').datum(this);
```

Comment:
* needs to be `.datum(thisChart)`
* remove both `this.wrap.datum(this)`s in `src/charts/init.js`
* remove `this.wrap.datum(data)` in `src/charts/draw.js`
* attach table and controls objects to their respective containers

Reply:
Well, thats a lot of changes for one line of code! On it. 😄

Reply:
Forgive my terseness.  I noticed I couldn't get to the datum in the browser with `d3.select('.wc-chart').datum()`, nor in `onResize()`, and I was pretty confused for a minute.

Reply:
All good! Will get this cleaned up now.

Reply:
Re: attach table and controls objects to their respective containers

I think I'm going to leave this out in `table`s for now, since they have the data bound to the wrap, and it seems like it might be used in the transform data function ... Will create an issue to deal with it down the road (possibly in v2.0).